app/controllers/ops_controller/diagnostics.rb

```diff
@@ -420,7 +420,7 @@ def diagnostics_server_list
       zone = Zone.find_by_id(x_node.split("-").last)
       @view, @pages = get_view(MiqServer, :named_scope => [[:with_zone_id, zone.id]]) # Get the records (into a view) and the paginator
     else
-      @view, @pages = get_view(MiqServer) # Get the records (into a view) and the paginator
+      @view, @pages = get_view(MiqServer, :named_scope => [[:in_region, MiqRegion.my_region.region]]) # Get the records (into a view) and the paginator
```

Comment:
great! and I believe can also use `:in_my_region` instead `:in_region`👍

Reply:
Yeah that works too and looks better. Changed 👍